client/a8c-for-agencies/sections/marketplace/checkout/style.scss

```diff
@@ -244,8 +233,7 @@
 	}
 
 	.product-info__label {
-		font-size: rem(16px);
-		font-weight: 700;
+		@include a4a-font-heading-md;
```

Comment:
`font-weight` here is 700, and in the `a4a-font-heading-md` it is 500. Is that ok?